src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -343,6 +361,67 @@ public BiomeGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 		pane5.add("Center", PanelUtils.totalCenterInPanel(spawnproperties));
 
+
+
+		JPanel multiproperties = new JPanel(new GridLayout(8, 3, 0, 2));
+		multiproperties.setOpaque(false);
+
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/use_pure_multi_noise"),
+				L10N.label("elementgui.biome.use_pure_multi_noise")));
+		multiproperties.add(usePureMultiNoise);
+		multiproperties.add(L10N.label("elementgui.biome.multi_noise_common_empty"));

```

Comment:
This is not the proper way to add a blank space.

Reply:
How should I do it?

Reply:
Use 2 columns instead of 3 and you won't need these empty labels.

Reply:
@Defeatomizer  but what about other rows?
![image](https://user-images.githubusercontent.com/46889499/177388387-c6cdd0b4-7334-4b58-9cb1-d9131ff5cac6.png)

Should I do something like subpanes as I saw in other mod element GUIs?

Reply:
I would also remove checkbox but only directly transform values for both sections

Reply:
@KlemenDEV
What do you mean by "directly transform values for both sections"? I dont really undestand what you mean

Reply:
If you enter ranges for noise values, it should "estimate" temperature, humidity, etc. and if you enter eg. humidity, it should recalculate multinoise values

Reply:
@KlemenDEV oh I get it now. I can try doing that, but I think that needs to wait till the 1.16 is dropped as you said.
Also would you preffer merging these 2 sections into one?
![image](https://user-images.githubusercontent.com/46889499/177397835-ff33a96f-93c1-475d-bcb1-b159d0d63ab2.png)

Also, should I wait until 1.16 drop or should I continue the work on it now?

Reply:
> , should I wait until 1.16 drop or should I continue the work on it now?

This is up to you, but also not related to the thread in this discussion

> Also would you preffer merging these 2 sections into one?

Yes